relatedRemittance.go

```diff
@@ -177,6 +180,18 @@ func (rr *RelatedRemittance) Validate() error {
 	if err := rr.isAlphanumeric(rr.RemittanceData.AddressLineSeven); err != nil {
 		return fieldError("AddressLineSeven", err, rr.RemittanceData.AddressLineSeven)
 	}
+	if err := rr.isAlphanumeric(rr.RemittanceData.CountryOfResidence); err != nil {
+		return fieldError("CountryOfResidence", err, rr.RemittanceData.CountryOfResidence)
+	}
+	return nil
+}
+
+// fieldInclusion validate mandatory fields. If fields are
+// invalid the WIRE will return an error.
+func (rr *RelatedRemittance) fieldInclusion() error {
```

Comment:
Nice find!

Reply:
Thanks! There were a few bugs like this that were uncovered by unwrapping test assertions from conditional blocks.